src/fdb5/api/FDB.h

```diff
@@ -149,6 +154,8 @@ class FDB {
     bool reportStats_;
 
     FDBStats stats_;
+
+    ArchiveCallback callback_ = nullptr;
```

Comment:
Why is this both here and in FDBBase. It feels like it should belong there more cleanly.

Reply:
Likely a mistake